Gemfile

```diff
@@ -15,22 +15,9 @@ gem 'cgi', '~> 0.3.6'
 # see https://github.com/ruby/set/pull/2
 gem 'sorted_set'
 
-# Force HTTPS for github-source gems.
-# This is a temporary workaround - remove when bundler version is >=2.0
-# @see https://github.com/bundler/bundler/issues/4978
-git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
-  "https://github.com/#{repo_name}.git"
-end
-
 gem 'rails', '~> 6.1'
 gem 'rails-controller-testing', '~> 1.0.5'
 
-# Compile Sprockets assets concurrently in `assets:precompile`.
-# Ref: https://github.com/rails/sprockets/pull/470
-gem 'sprockets', github: 'code-dot-org/sprockets', ref: 'concurrent_asset_bundle_3.x'
-gem 'sprockets-rails', '3.3.0'
```

Comment:
This was breaking things, and its a back-port of a Rails 4 feature to Rails 3 that got us stuck on Rails 3 sprockets for N years.

Long-term maintenance lesson: KISS, stay with upstream, clever short term fixes tend to become long-term breakage. Optimize for 10 years.